src/commands/lightning/dev/app.ts

```diff
@@ -101,6 +101,11 @@ export default class LightningDevApp extends SfCommand<void> {
       return Promise.reject(new Error(messages.getMessage('error.username')));
     }
 
+    const localDevEnabled = await OrgUtils.isLocalDevEnabled(connection);
```

Comment:
@nrkruk suggested that this should be applied across the plugin, to cover the `site` command as well. Should we add that check into this PR?

Reply:
Oh I wasn't aware of this. Sure I can add it there too.